### ISSUE TITLE
Wrong labels

### ISSUE THREAD

**Issue by Cuberick-Orion** [erobic/faster-rcnn.pytorch#2]
I think the labels corresponding to the classification is wrong/

**Comment by erobic** [comment]
Hi, the model was pre-trained using classes based on all attributes.

Could you provide some examples as to where the labels are wrong?
